test/blocks/shared/constants.test.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from '@esm-bundle/chai';
+import { getDaAdmin, COLLAB_ORIGIN } from '../../../blocks/shared/constants.js';
+
+import '../../milo.js';
+
+describe('DA Admin', () => {
+  describe('Set DA admin default', () => {
```

Comment:
More a question than a comment... I thought that the actual Mocha tests were normally put in `it(..)` calls while the test suite sections/groupings are in `describe(...)` calls. I see that you have the actual tests also in `describe` calls. Any particular reason for this?

Reply:
You're right. Bad copy / paste. It should be `it`. Will fix. Thanks @bosschaert!

Reply:
@bosschaert this should be fixed now. Thanks for reviewing.